pandas/tests/indexes/multi/test_analytics.py

```diff
@@ -272,50 +281,28 @@ def test_map_dictlike(idx, mapper):
     np.arccos, np.arctan, np.sinh, np.cosh, np.tanh,
     np.arcsinh, np.arccosh, np.arctanh, np.deg2rad,
     np.rad2deg
-])
-def test_numpy_ufuncs(func):
+], ids=lambda func: func.__name__)
+def test_numpy_ufuncs(idx, func):
     # test ufuncs of numpy. see:
     # http://docs.scipy.org/doc/numpy/reference/ufuncs.html
 
```

Comment:
I think this broke some PY2 tests: https://api.travis-ci.org/v3/job/489376151/log.txt (ok to just skip them for now)

Reply:
https://travis-ci.org/MacPython/pandas-wheels/builds/489376149

Reply:
on it.